zulip_bots/zulip_bots/bots/wikipedia/test_wikipedia.py

```diff
@@ -47,9 +47,10 @@ def test_bot(self):
             )
 
         # Empty query, no request made to the Internet.
-        bot_response = "Please enter your message after @mention-bot"
-        self.assert_bot_response(
-            message = {'content': ''},
-            response = {'content': bot_response},
-            expected_method='send_reply'
-        )
+        bot_response = "Please enter your message after @foobot"
+        with self.mock_bot_identity("foobot"):
```

Comment:
I think for the tests, we should just have some fixed value, `@test-bot` be the mocked name of the bot in the test framework, and have that be used everywhere in the tests.  It feels really ugly to need every test to need to do this `with` statement.